Sources/PointFree/Account/PrivateRss.swift

```diff
@@ -159,11 +158,13 @@ private func fetchStripeSubscriptionForUser<A>(
     }
 }
 
-private let privateEpisodesFeedView = itunesRssFeedLayout <| View<(Stripe.Subscription?, User)> { subscription, user -> Node in
-  node(
-    rssChannel: privateRssChannel(user: user),
-    items: items(forUser: user, subscription: subscription)
-  )
+private let privateEpisodesFeedView = itunesRssFeedLayout { (data: (subscription: Stripe.Subscription?, user: User)) -> [Node] in
```

Comment:
oh interesting. I never thought about using this syntax for a layout. could we even replace the other `layout <<< { ... }`'s with just `layout { ... }` ?

Reply:
We'd need to un-curry `simpleEmailLayout`. Worth doing but let's kick to another PR.